src/fedora/36/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36
+
+# Install Helix Dependencies
+
+RUN dnf install -y \
+        openssl \
+        python3 \
+        python3-devel \
+        libatomic \
+        libffi-devel \
+        redhat-rpm-config \
+        sudo && \
```

Comment:
these are the Helix dependencies.  Stuff like openSSL, python, and sudo are absolutely needed but if you're certain that they're already there enough to risk the several hours putting them back, it's fine to remove them; as noted it will only impact this one image anyways.

Reply:
How about the pip & virtualenv? Can we get them via system packages instead of installing them directly?

Reply:
Pip's the package manager and it's not installed in this docker image by default:
``` 
C:\scratch>docker run -it registry.fedoraproject.org/fedora:36 bash
[root@5daa616fd751 /]# python3 -m pip
/usr/bin/python3: No module named pip
```
So yes, helix still needs this to be present for later install commands below.

Reply:
There is. I explicitly remove it bellow with `RUN dnf remove -y python3-pip && \` because it was conflating with the custom logic.